src/HotChocolate/Core/src/Types/Types/InputObjectType.cs

```diff
@@ -53,7 +53,7 @@ public bool IsInstanceOfType(IValueNode literal)
                 || literal is NullValueNode;
```

Comment:
- [ ] @michaelstaib we have to rework the serialization of input types before merging

Reply:
I can work on that if you can be more specific with what the issues are

Reply:
We will first have to discuss this .... will setup a meeting for this.